### DISCUSSION TITLE
actionAdminProductsListingFieldsModifier work in PS 1.7 but not in PrestaShop 8

### DISCUSSION THREAD

**Discussion by vipulhadiya** [PrestaShop/PrestaShop#36636] [Q&A]
I had developed a module to add custom column in admin products list. The module works like charm in PrestaShop 1.7, but it does not work in PS 8 anymore.

I used these hooks:
actionAdminProductsListingFieldsModifier
displayAdminCatalogTwigListingProductFields
displayAdminCatalogTwigProductFilter
displayAdminCatalogTwigProductHeader

I think these hooks are present in the PrestaShop 8 but still they are not working anymore.
Does anyone have/had the same issue?

**Comment by kpodemski** [comment]
PrestaShop 1.7 and PrestaShop 8 use different mechanisms to render the product list.

The first version of Symfony-base architecture is used in PrestaShop 1.7, and in PrestaShop 8, the product list is rendered using the [Grid component](https://devdocs.prestashop-project.org/8/development/components/grid/#body-inner).

You can, of course, [modify the Grid from the module](https://devdocs.prestashop-project.org/8/development/components/grid/tutorials/modify-grid-in-module/).
